includes/Elements/SVG_Draw.php

```diff
@@ -491,7 +491,7 @@ protected function register_controls() {
 
 	protected function render() {
 		$settings = $this->get_settings_for_display();
-		$svg_html = isset( $settings['svg_html'] ) ? preg_replace( '#<script(.*?)>(.*?)</script>#is', '', $settings['svg_html'] ) : '';
+		$svg_html = isset( $settings['svg_html'] ) ? preg_replace( ['#<script(.*?)>(.*?)</script>#is', '#<script(.*?)>(.*?)</script#is'], '', $settings['svg_html'] ) : '';
```

Comment:
Intensional change?

Reply:
yes. By this we remove script tags from client input, but here i notice if client input `<script>code</script` (didn't put `>` ) it executes. so i add this rules also.